applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -216,6 +245,79 @@ load_conference(DocId, Context) ->
 %%--------------------------------------------------------------------
 %% @private
 %% @doc
+%% Load details of the participants of a conference
+%% @end
+%%--------------------------------------------------------------------
+-spec load_conference_details(cb_context:context(), path_token()) -> cb_context:context().
+load_conference_details(Context, ConfId) ->
+    AccountRealm = wh_util:get_account_realm(cb_context:account_db(Context), cb_context:account_id(Context)),
+    Req = [{<<"Realm">>, AccountRealm}
+           ,{<<"Fields">>, []}
+           ,{<<"Conference-ID">>, ConfId}
+           | wh_api:default_headers(?APP_NAME, ?APP_VERSION)
+          ],
+    ReqResp = whapps_util:amqp_pool_collect(Req
+                                            ,fun wapi_conference:publish_search_req/1
+                                            ,{'ecallmgr', 'true'}
+                                           ),
+    case ReqResp of
+        {'error', _} -> cb_context:add_system_error('conference_search_error', Context);
+        {_, JObjs} -> conference_participants(JObjs, Context)
+    end.
+
+-spec conference_participants(wh_json:objects(), cb_context:context()) -> cb_context:context().
```

Comment:
I don't like this whole function. Could you rewrite that as a simple list comprehension?